staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -235,76 +248,11 @@ func WithTypeAndObjectMeta(s *schema.Structural) *schema.Structural {
 // size, this can be used as an estimate as the worst case number of times data matching the schema could be repeated.
 // Note that this only assumes a single comma between data elements, so if the schema is contained under only maps,
 // this estimates a higher cardinality that would be possible.
-func MaxCardinality(s *schema.Structural) uint64 {
-	sz := estimateMinSizeJSON(s) + 1 // assume at least one comma between elements
+func MaxCardinality(minSize int64) uint64 {
```

Comment:
Also, since the only reasonable way to get the minSize is from a DeclType, document that fact in the godoc?  (or just take the DeclType as the param..)

Reply:
Whoops, I've fixed the godoc. I thought about passing the entire DeclType, but since `MinSerializedSize` is the only part of it that's used, I left in just `minSize` as a parameter.